nexus/tests/output/uncovered-authz-endpoints.txt

```diff
@@ -1,10 +1,7 @@
 API endpoints with no coverage in authz tests:
 probe_delete                             (delete "/experimental/v1/probes/{probe}")
-support_bundle_delete                    (delete "/experimental/v1/system/support-bundles/{support_bundle}")
 probe_list                               (get    "/experimental/v1/probes")
 probe_view                               (get    "/experimental/v1/probes/{probe}")
-support_bundle_list                      (get    "/experimental/v1/system/support-bundles")
-support_bundle_view                      (get    "/experimental/v1/system/support-bundles/{support_bundle}")
 support_bundle_download                  (get    "/experimental/v1/system/support-bundles/{support_bundle}/download")
 support_bundle_download_file             (get    "/experimental/v1/system/support-bundles/{support_bundle}/download/{file}")
 support_bundle_index                     (get    "/experimental/v1/system/support-bundles/{support_bundle}/index")
```

Comment:
It's probably possible to test these endpoints in the unathorized.rs test, but it's kinda a pain -- one of the "setup requests" there involves POST-ing the support bundle, but it doesn't wait for the background task of collection to complete asynchronously.